### PR TITLE
Update beats framework to dc74607

### PR DESCRIPTION
**Pull request by roncohen**

### CODE REVIEW

File: _beats/testing/environments/latest.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 version: '2.1'
 services:
   elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:6.0.0-rc1
+    image: docker.elastic.co/elasticsearch/elasticsearch:6.0.0-rc2
```
Comment (simitt) [inline _beats/testing/environments/latest.yml]: Shouldn't this be `6.0.0` now?

File: _beats/libbeat/docs/version.asciidoc [hunk 2]
```diff
@@ -1,6 +1,6 @@
 :stack-version: 6.1.0
```
Comment (simitt) [inline _beats/libbeat/docs/version.asciidoc]: I think this should be `6.0.1`?

Comment (roncohen) [reply _beats/libbeat/docs/version.asciidoc]: We have our own, that also needs to get updated: https://github.com/elastic/apm-server/blob/6.0/docs/copied-from-beats/version.asciidoc

File: _beats/testing/environments/latest.yml (hunk 1 above)
Comment (roncohen) [reply _beats/testing/environments/latest.yml]: Good find: https://github.com/elastic/beats/pull/5648

**Pull request merged**